test/kpod_inspect.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="alpine:latest"
```

Comment:
This value doesn't get used in the file.  We could either set it to `redis:alpine` and replace direct references to that image, or drop the variable.